WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -483,6 +483,7 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
     }
 
     func toggleEditingMode() {
+        emitPostSaveEvent()
```

Comment:
I believe emitting save event is warranted here as the app needs to have a "snapshot" of Gutenberg's content before toggling to a different mode.

Reply:
I'm hesitant about sending this event here because we're not strictly saving the post. I understand it solves the problem related to not saving the VideoPress block settings, but I'm concerned that in the future, we use this event for other purposes and this introduces an undesired side effect. WDYT?

Reply:
Following our discussion on this, and the discovery that this is an issue on the web, I've gone ahead to revert this change in https://github.com/wordpress-mobile/WordPress-iOS/pull/20571/commits/c653367c6cb89688bd58cf238dc6b3d51bb9857d.